cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2025,24 +2030,9 @@ def test_obsm_values_spatial_embedding_missing__is_single_true(self, validator_w
             "ERROR: 'spatial' embedding is required in 'adata.obsm' if adata.uns['spatial']['is_single'] is True."
         ]
 
-    def test_obsm_values_spatial_embedding_missing__is_single_false(self, validator_with_visium_assay):
-        validator = validator_with_visium_assay
-        validator.adata.uns["spatial"] = {"is_single": False}
+    def test_obsm_values_spatial_embedding_missing__is_single_false(self, validator_with_spatial_and_is_single_false):
+        validator = validator_with_spatial_and_is_single_false
         del validator.adata.obsm["spatial"]
-        # Format adata.obs into valid shape for Visium and is_single False.
-        validator.adata.obs.pop("array_col")
-        validator.adata.obs.pop("array_row")
-        validator.adata.obs.pop("in_tissue")
-        validator.validate_adata()
-        assert validator.errors == []
-
-    def test_obsm_values_spatial_embedding_present__is_single_false(self, validator_with_visium_assay):
```

Comment:
moved this test into `test_validate.py` as `test__validate_spatial_is_single_false_ok` and renamed now that we have the fixture `adata_spatial_is_single_false` with spatial embedding present